src/main/java/com/example/bookwonders/controller/ShoppingCartController.java

```diff
@@ -34,19 +38,25 @@ public ShoppingCartResponseDto getCart() {
     }
 
     @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
     @Operation(summary = "Add book to shopping cart")
-    public ShoppingCartResponseDto addBookToCart(AddCartItemRequestDto requestDto) {
+    public ShoppingCartResponseDto addBookToCart(
+            @RequestBody @Valid AddCartItemRequestDto requestDto
+    ) {
         return shoppingCartService.addCartItem(requestDto);
     }
 
     @PutMapping("/cart-items/{cartItemId}")
     @Operation(summary = "Update quantity of book in shopping cart")
-    public ShoppingCartResponseDto updateQuantityOfBook(@PathVariable Long cartItemId,
-                                                        UpdateBookQuantityInCartDto requestDto) {
+    public ShoppingCartResponseDto updateQuantityOfBook(
+            @PathVariable Long cartItemId,
+            @RequestBody @Valid UpdateBookQuantityInCartDto requestDto
+    ) {
```

Comment:
```suggestion
            @RequestBody @Valid UpdateBookQuantityInCartDto requestDto) {
```
mate is using google style